fonts/other/uncut-sans/files/rename.py

```diff
@@ -0,0 +1,48 @@
+import os 
```

Comment:
Looks good to me! I just think we need to remove this one file and we should be good to go!